src/containers/profile/ProfileSections/AccountSettings/AccountSettingsSections/ProfileInfo/ProfileInfoSections/ProfileInfoSections.css

```diff
@@ -0,0 +1,14 @@
+.infoSignImg {
+  width: 20px;
+}
+
+.infoContainer > * {
```

Comment:
using `> *` is generally not recommended. for the font-size and color, you don't need this. for the margin-bottom, can you use a more specific selector please for what you want?